### ISSUE TITLE
Increase campaign name length limits from 100 characters

### ISSUE THREAD

**Issue by jlev** [OpenSourceActivismTech/call-power#109]
https://sentry.io/organizations/spacedog/issues/1027832007/?referrer=github_plugin

```
DataError: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely) (psycopg2.DataError) value too long for type character varying(100)
 [SQL: 'INSERT INTO campaign_campaign (created_time, name, country_code, campaign_type, campaign_state, campaign_subtype, campaign_language, segment_by, locate_by, include_special, target_ordering, target_shuffle_chamber, target_offices, call_maximum, allow_call_in, allow_intl_calls, prompt_schedule, status_code, embed) VALUES (%(created_time)s, %(name)s, %(country_code)s, %(campaign_type)s, %(campaign_state)s, %(campaign_subtype)s, %(campaign_language)s, %(segment_by)s, %(locate_by)s, %(include_special)s, %(target_ordering)s, %(target_shuffle_chamber)s, %(target_offices)s, %(call_maximum)s, %(allow_call_in)s, %(allow_intl_calls)s, %(prompt_schedule)s, %(status_code)s, %(embed)s) RETURNING campaign_campaign.id'] [parameters: {'allow_intl_calls': False, 'locate_by': u'None', 'created_time': datetime.datetime(2019, 5, 15, 14, 40, 17, 946533), 'campaign_type': u'custom', 'include_special': u'', 'embed': 'null', 'status_code': 1, 'country_code': u'us', 'call_maximum': None, 'campaign_language': u'en', 'target_ordering': u'shuffle', 'campaign_subtype': u'None', 'segment_by': u'custom', 'allow_call_in': True, 'prompt_schedule': False, 'campaign_state': u'None', 'name': u'Calls needed: Demand that Speaker Pelosi and Rep. Nadler investigate and impeach Trump immediately (MCommons)', 'target_offices': u'', 'target_shuffle_chamber': True}]
(27 additional frame(s) were not displayed)
...
  File "call_server/campaign/views.py", line 163, in form
    campaign_target = CampaignTarget.query.filter_by(campaign=campaign, target=target).one()

DataError: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely) (psycopg2.DataError) value too long for type character varying(100)
 [SQL: 'INSERT INTO campaign_campaign (created_time, name, country_code, campaign_type, campaign_state, campaign_subtype, campaign_language, segment_by, locate_by, include_special, target_ordering, target_shuffle_chamber, target_offices, call_maximum, allow_call_in, allow_intl_calls, prompt_schedule, status_code, embed) VALUES (%(created_time)s, %(name)s, %(country_code)s, %(campaign_type)s, %(campaign_state)s, %(campaign_subtype)s, %(campaign_language)s, %(segment_by)s, %(locate_by)s, %(include_special)s, %(target_ordering)s, %(target_shuffle_chamber)s, %(target_offices)s, %(call_maximum)s, %(allow_call_in)s, %(allow_intl_calls)s, %(prompt_schedule)s, %(status_code)s, %(embed)s) RETURNING campaign_campaign.id'] [parameters: {'allow_intl_calls': False, 'locate_by': u'None', 'created_time': datetime.datetime(2019, 5, 15, 14, 40, 17, 946533), 'campaign_type': u'custom', 'include_special': u'', 'embed': 'null', 'status_code': 1, 'country_code': u'us', 'call_maximum': None, 'campaign_language': u'en', 'target_ordering': u'shuffle', 'campaign_subtype': u'None', 'segment_by': u'custom', 'allow_call_in': True, 'prompt_schedule': False, 'campaign_state': u'None', 'name': u'Calls needed: Demand that Speaker Pelosi and Rep. Nadler investigate and impeach Trump immediately (MCommons)', 'target_offices': u'', 'target_shuffle_chamber': True}]
```

**Comment by jlev** [comment]
Additionally, enforce the limit in the frontend so campaign users don't see the DataError when they try to save.

**Comment by jlev** [comment]
done with https://github.com/OpenSourceActivismTech/call-power/tree/bugfix/campaign-name-max-length

**Comment by jlev** [comment]
released v1.5
